File: BackEnd/LolWebApiRest/LolWebApiRest/Models/Duo.cs
namespace LolWebAPI.Models
{
    public class Duo
    {

        public int Id { get; set; }
        public string Puuid { get; set; }
        public string GameName { get; set; }
        public string TagLine { get; set; } 
        public int IdLane { get; set; }
        public int IdLaneDuo { get; set; }
        public int IdElo { get; set; }
        public int IdModoDeJogo { get; set; }
        public bool IsVoiceUser { get; set; }
        public string Note { get; set; }
    }
}

File: LolWebSite/src/app/models/duo.ts
import { AccountInformations } from "./account copy";

export interface Duo {
    id : number ;
    puuid : string ;
    gameName : string ;
    tagLine : string ; 
    lane : string ;
    laneDuo : string ;
    elo : string ;
    modoDeJogo : string ;
    isVoiceUser : boolean ;
    note : string ;
    idIcone: number;
    accountInformations : AccountInformations;
}

